client/starwhale/utils/debug.py

```diff
@@ -10,17 +10,36 @@
 )
 
 
-def init_logger(verbose: int) -> None:
+def init_logger(verbose: int = 0) -> None:
+    """Initialize Starwhale logger and traceback.
+
+    Arguments:
+        verbose: (int, optional) verbosity level. Defaults to 0.
+          - 0: show only errors, traceback only shows 1 frame.
+          - 1: show errors + warnings, traceback shows 5 frames.
+          - 2: show errors + warnings + info, traceback shows 10 frames.
+          - 3: show errors + warnings + info + debug, traceback shows 100 frames.
+          - >=4: show errors + warnings + info + debug + trace, traceback shows 1000 frames.
+
+    Returns: None
+    """
     if verbose == 0:
         lvl = console.ERROR
+        max_frames = 1
```

Comment:
It seems that the max `1` has no effect
ref: https://github.com/Textualize/rich/blob/8c7449f987a5c423a162aacdf969d647e6085918/rich/traceback.py#L290